framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageChannelFactory.cs

```diff
@@ -0,0 +1,117 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.Collections.ObjectModel;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+    /// <summary>
+    /// IChannelFactory implementation for AzureQueueStorage.
+    /// </summary>
+    class AzureQueueStorageChannelFactory : ChannelFactoryBase<IOutputChannel>
+    {
+        #region member_variables
+        BufferManager bufferManager;
+        MessageEncoderFactory messageEncoderFactory;
+        bool multicast;
+        #endregion
+
+        /*internal UdpChannelFactory(UdpTransportBindingElement bindingElement, BindingContext context)
+            : base(context.Binding)
```

Comment:
Without a constructor, how are you populating messageEncoderFactory?

Reply:
Thats my bad, There was a line in that piece of code which was not compiling, so i commented out to visit it later. updated it now.